docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -88,7 +90,7 @@ Configure the `azure-cli` repository information as shown previously. Available
     AZ_VER=2.51.0
 
     # Install a specific version
-    sudo apt-get install azure-cli=$AZ_VER-1~$AZ_DIST
+    sudo apt-get install azure-cli=${AZ_VER}-1~${AZ_DIST}
```

Comment:
I added thecurly brackets since `-` is not part of a variable name.
It might not be need it this particular example, but recommended to minimize unexpected things happening.